### PR TITLE
message queue - jenkinsfileTrigger

### PR DESCRIPTION
**Pull request by joejstuart**

### CODE REVIEW

File: JenkinsfileTrigger [hunk 1]
```diff
@@ -54,35 +54,47 @@ node() {
...
                     validMessage = packagepipelineUtils.checkBranch()
 
-
                 }
 
             }
 
             if (validMessage) {
+
+                messageFields = packapgepipelineUtils.setMessageFields('package.queued')
```
Comment (johnbieren) [inline JenkinsfileTrigger]: Looks like you copy pasted my typo =)

File: JenkinsfileTrigger [hunk 2]
```diff
@@ -54,35 +54,47 @@ node() {
...
+                    } catch(Throwable err) {
+                        currentBuild.description = "*TRIGGER FAILURE*"
+                        error "Error: Build could not be added to queue after " + TRIGGER_RETRY_COUNT + " tries"
+                        throw err
                     }
                 }
 
             } else {
                 echo "CI_MESSAGE was invalid. Skipping..."
+                messageFields = packapgepipelineUtils.setMessageFields('package.ignored')
```
Comment (johnbieren) [inline JenkinsfileTrigger]: typo

File: JenkinsfileTrigger (hunk 1 above)
Comment (joejstuart) [reply JenkinsfileTrigger]: lol. Probably did.

File: JenkinsfileTrigger (hunk 2 above)
Comment (joejstuart) [reply JenkinsfileTrigger]: fixed

**Pull request merged**